tsconfig.base.json

```diff
@@ -0,0 +1,45 @@
+{
+  // This tsconfig file contains the shared config for the build (tsconfig.build.json) and type checking (tsconfig.json) config.
```

Comment:
comments are not allowed in JSON files, are they?

Reply:
I have no issues with them as part of tsconfig. Do you?